flask_classful.py

```diff
@@ -180,6 +180,21 @@ def make_proxy_method(cls, name):
         i = cls()
         view = getattr(i, name)
 
+        # Since the view is a bound instance method, first make it an actual function
```

Comment:
@hoatle Let me know what you think.  I think this is a better solution than moving the decorators to the bottom. It will allow the decorator list to be applied to all memebers (get, put, index, etc) *then* the proxy is generated.  Wrapping the view as a function also allows for decorators to apply attributes.